docs/ai-core-APIs.md

```diff
@@ -0,0 +1,213 @@
+# Available APIs
+
+✅ Available and tested
+
+❗Warning: The APIs not marked as tested in this section are in an experimental state. Use them at your own risk.
+
+# AI Core
+
+- ApplicationAPI
+- ArtifactApi ✅
+- ConfigurationApi ✅
+- DeploymentApi ✅
+- DockerRegistrySecretApi
+- ExecutableApi
+- ExecutionApi ✅
+- ExecutionScheduleApi
+- FileApi
+- KpiApi
+- MetaApi
+- MetricsApi
+- ObjectStoreSecretApi
+- RepositoryApi
+- ResourceApi
+- ResourceGroupApi
+- ResourceQuotaApi
+- ScenarioApi ✅
+- SecretApi
+- ServiceApi
+
+## ArtifactApi
```

Comment:
[req] Consider linking the pre-requisites here(Enabling AI Core service in BTP), before delving into the usage.